Lib/test/test_support.py

```diff
@@ -161,6 +161,20 @@ def test_temp_dir__existing_dir__quiet_true(self):
                                         f'temporary directory {path!r}: '),
                         warn)
 
+    @unittest.skipUnless(hasattr(os, "fork"), "test requires os.fork")
+    def test_temp_dir__forked_child(self):
+        """Test that a forked child process does not remove the directory."""
+        with support.temp_cwd() as temp_path:
```

Comment:
Hum, please don't use fork in a test process. Write a short script run with test.support.script_helper. Otherwise, I expect too many issues like bad handling of KeyboardInterrupt. Use textwrap.dedent() to include the script directly here.